CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.15.3)
+
+project(XMLCoder LANGUAGES Swift)
+
+set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+if(CMAKE_SYSTEM_NAME STREQUAL Windows OR CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  option(BUILD_SHARED_LIBS "Build shared libraries by default" YES)
+endif()
+
+add_library(XMLCoder 
+    Sources/XMLCoder/Auxiliaries/Box/BoolBox.swift

```

Comment:
Would there be an easy way to recursively add all `.swift` fies in `Sources/XMLCoder` directory and its subdirectories? I'm a bit afraid of people forgetting to update this list after adding a file or two to the project.

Reply:
OK. I will add the ability to recursively capture source files via GLOB_RECURSE

Reply:
Fixed